conf.py

```diff
@@ -83,42 +83,47 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ["*/build-*",
+exclude_patterns = ["**/build-*",
                     ".venv",
-                    "atmel-samd/asf",
-                    "atmel-samd/asf_conf",
-                    "atmel-samd/common-hal",
-                    "atmel-samd/boards",
-                    "atmel-samd/QTouch",
-                    "atmel-samd/*.c",
-                    "atmel-samd/*.h",
-                    "bare-arm",
-                    "cc3200",
-                    "cc3200/FreeRTOS",
-                    "cc3200/hal",
                     "drivers",
-                    "esp8266",
                     "examples",
                     "extmod",
                     "frozen",
                     "lib",
-                    "minimal",
                     "mpy-cross",
-                    "pic16bit",
+                    "ports/*/*.c",
+                    "ports/*/*.h",
+                    "ports/*/boards",
+                    "ports/*/common-hal",
+                    "ports/*/supervisor",
+                    "ports/atmel-samd/asf4",
+                    "ports/atmel-samd/asf4_conf",
+                    "ports/atmel-samd/QTouch",
+                    "ports/bare-arm",
+                    "ports/cc3200",
+                    "ports/cc3200/FreeRTOS",
+                    "ports/cc3200/hal",
+                    "ports/esp8266",
+                    "ports/minimal",
+                    "ports/nrf/devices",
```

Comment:
I think it should be `device` not `devices`

Reply:
updated!